contracts/arbos/builtin/ArbAggregator.sol

```diff
@@ -25,5 +25,13 @@ interface ArbAggregator {
     // This reverts unless called by the address that would be returned by getFeeCollector(aggregator),
     //      or by the chain owner.
     function setFeeCollector(address aggregator, address newFeeCollector) external;
+
+    // Get the tx base fee (in approximate L1 gas) for aggregator
+    function getTxBaseFee(address aggregator) external view returns (uint);
+
+    // Set the tx base fee (in approximate L1 gas) for aggregator
+    // If feeInL1Gas is outside the chain's allowed bounds, fee will be "clipped" to the min or max allowed value
+    // Reverts unless called by aggregator or the chain owner
+    function setTxBaseFee(address aggregator, uint feeInL1Gas) external;
```

Comment:
Personally I would prefer reverting on bad inputs (out of allowed range) rather than silently adjusting. Would probably be a good question to raise with the group

Reply:
Added an argument to let the caller choose which behavior they want on out-of-bound value: revert or clip.